src/nomad_simulations/schema_packages/properties/__init__.py

```diff
@@ -18,7 +18,33 @@
 
 from .band_gap import ElectronicBandGap
 from .band_structure import ElectronicBandStructure, ElectronicEigenvalues
-from .energies import ChemicalPotential, FermiLevel
+from .energies import (
+    FermiLevel,
+    BaseEnergy,
+    EnergyContribution,
+    TotalEnergy,
+    KineticEnergy,
+    PotentialEnergy,
+)
+from .forces import BaseForce, ForceContribution, TotalForce
+from .thermodynamics import (
+    Pressure,
+    Volume,
+    Temperature,
+    HeatAdded,
```

Comment:
Shoudn't be `Heat` and `Work`? The sign of `value` should establish the directionality anyways. You can even add a simplified string to indicate this via normalization, e.g.:
```python
class Work(PhysicalProperty)
    type = Quantity(type=MEnum('done-to-system', 'done-by-system')

    def resolve_type(self):
         if self.value < 0:
              return 'done-by-system'
         elif self.value > 0:
              return 'done-to-system'
```

In any case, I remember Chemistry and Physics always conflicting on the signs for heat and work. Hopefully we can agree on not defining `PhysicalWork` and `ChemicalWork`...... 😆

Reply:
I think you have to specify somehow in the name or definition what your sign convention is. As you say, Chemistry and Physics typically do this in an opposite manner. I figured that if it's in the name it would be less prone to mistakes, but we could also only put it in the description. 

Let's maybe leave this for the taxonomy, I will just change to Heat and Work now and then add the sign convention in the description